_posts/2020-10-04-quarkus-native-on-aarch64.adoc

```diff
@@ -0,0 +1,408 @@
+---
+layout: post
+title: 'Quarkus native running on aarch64'
```

Comment:
```suggestion
title: 'Quarkus native running on a Raspberry Pi'
```
I would put the much more known Raspberry name in the title rather than the technical correct but cryptic name `aarch64`.

Reply:
I agree with your concern. Still, even tho "aarch64" is too generic, I'd rather change "aarch64" with "ARMv8" to underline the compliance and supportability of that CPU family. Anyway, if you believe it's better to replace "aarch64" with "Raspberry Pi" I'm more than glad to commit your suggestion.

Reply:
For attention mention raspberry seems more likely. 

Could add "or other ArmV8" to.be complete :)

Reply:
Thanks for your reply. I'll merge your suggestion